apps/nextjs-website/src/pages/[productSlug]/overview.tsx

```diff
@@ -31,10 +33,12 @@ export const getStaticProps: GetStaticProps<OverviewPageProps, Params> = ({
   }
 };
 
-const OverviewPage = ({ hero }: OverviewPageProps) => (
-  <>
-    <Hero title={hero?.title || 'missing title'} subtitle={hero?.subtitle} />
-  </>
-);
+const OverviewPage = ({ hero, product, path }: OverviewPageProps) => {
+  return (
+    <Layout currentProduct={product} currentSlug={path}>
```

Comment:
You right, i had use `slug` instead of `path` to be consistent with naming that was used before align branch to main now i have update the name also here.